skimage/measure/_find_contours_cy.pyx

```diff
@@ -13,14 +18,17 @@ cdef inline double _get_fraction(double from_value, double to_value,
 
 
 def iterate_and_store(double[:, :] array,
-                      double level, Py_ssize_t vertex_connect_high):
+                      double level, Py_ssize_t vertex_connect_high,
+                      np.ndarray[DTYPE_BOOL_t, cast=True, ndim=2] mask):
```

Comment:
This is the first time I encountered this syntax. Is there a specific reason/advantage over using a memoryview like `np.uint8_t[:, :] mask` here?

Reply:
The [official docs](https://cython.readthedocs.io/en/latest/src/tutorial/numpy.html) seem to recommend using memoryviews for their performance advantage.

Reply:
I'm new to Cython, so I appreciate the help here! `np.uint8_t[:, :]` was what I tried first, but I ran into the error `ValueError: Does not understand character buffer dtype format string`, and this approach was what I found googling for help.

Since iterate_and_store isn't a public API anyway, I guess the right solution is to make it take a uint8 memoryview as you suggest, and cast mask to `dtype=np.uint8` in `find_contours`?